Explorer/Assets/Plugins/CommandTerminal/LocalSceneTerminal.cs

```diff
@@ -0,0 +1,214 @@
+using UnityEngine;
+using UnityEngine.Assertions;
+
+// ORIGINAL OPEN SOURCE PLUGIN SCRIPT:
+// https://github.com/stillwwater/command_terminal/blob/0f5918ea79014955b24c7d431625a97a1cac8797/CommandTerminal/Terminal.cs
+
+namespace CommandTerminal
+{
+    public enum TerminalState
+    {
+        Close,
+        OpenSmall,
+        OpenFull,
+    }
+
+    /// <summary>
+    ///     Dedicated toggleable console for displaying only SDK scene messages during local scene development
+    /// </summary>
+    public class LocalSceneTerminal : MonoBehaviour
+    {
+        private const string ESCAPE_KEYBOARD_EVENT = "escape";
+
+        [Header("Window")]
+        [Range(0, 1)]
+        [SerializeField]
+        private float maxHeight = 0.7f;
+
+        [SerializeField]
+        [Range(0, 1)]
+        private float smallTerminalRatio = 0.33f;
+
+        [Range(100, 1000)]
+        [SerializeField]
+        private float toggleSpeed = 1000;
+
+        [SerializeField] private string toggleHotkey = "`";
+        [SerializeField] private string toggleFullHotkey = "#`";
+        [SerializeField] private int logsMaxAmount = 512;
+
+        [Header("Theme")]
+        [SerializeField] private int fontSize = 20;
+        [SerializeField] private Font consoleFont;
+        [SerializeField] private Color backgroundColor = Color.black;
+        [SerializeField] private Color foregroundColor = Color.white;
+        [SerializeField] private Color warningColor = Color.yellow;
+        [SerializeField] private Color errorColor = Color.red;
+        private Texture2D backgroundTex;
+        private ConsoleBuffer consoleBuffer;
+        private float currentOpenValue;
+        private TextEditor editorState;
+        private GUIStyle labelStyle;
+        private float openTarget;
+        private float realWindowSize;
+        private Vector2 scrollPosition;
+        private TerminalState state;
+        private Rect window;
+        private GUIStyle windowStyle;
+
+        private bool isClosed => state == TerminalState.Close && Mathf.Approximately(currentOpenValue, openTarget);
+
+        private void Start()
+        {
+            Assert.AreNotEqual(toggleHotkey.ToLower(), "return", "Return is not a valid ToggleHotkey");
```

Comment:
why do we have a validation only for return? shouldn't we validate more "reserved" keys?

Reply:
Good point, I'll remove the validation altogether for now and we can add a more defined validation in the future if we really need it, since this is configured by us and not by creators.

Reply:
Good point, I'll remove the validation algotether. We control this setup and not creators so if in the future we decide we need the validation we add whatever we want.

Implemented change at: https://github.com/decentraland/unity-explorer/pull/1546/commits/7db392aa7c131317dd50483f83fe60edbcb8d882